#!/usr/bin/env python
PACKAGE = "vision"

from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

# Params for line task
gen.add("loOrange", int_t, 0, "Low L bound for Orange", 143, 0, 255)
gen.add("areaThresh", int_t, 0, "Low V bound for area", 50, 0, 2000)
gen.add("hiOrange", int_t, 0, "High L bound for Orange", 190, 0, 255)
gen.add("upperAreaThresh", int_t, 0, "High V bound for area", 191, 0, 25500)
#gen.add("upperThresh",int_t,0,"Minimum threshold area for buoy contours",250,0,1000)
#gen.add("blur",bool_t,0,"True to enable 5x5 Gaussian blur ",True)
gen.add("blur",bool_t,0,"Apply blur to filter chain",False)


exit(gen.generate(PACKAGE, "line_task", "line"))
